corehq/apps/app_manager/static/app_manager/js/manage_releases_by_app_profile.js

```diff
@@ -26,6 +26,16 @@ hqDefine('app_manager/js/manage_releases_by_app_profile', [
             self.appName = details.app_name;
             self.version = details.version;
             self.profileName = details.profile_name;
+            var showHostedCCZLink = initialPageData.get("show_hosted_cczs_link");
```

Comment:
What would you think of checking this on the client side, using `hqImport('hqwebapp/js/toggles').toggleEnabled('MANAGE_CCZ_HOSTING')`? It'd be more direct/concise than passing this flag through initial page data.

Reply:
just confirming that this does not send a new request and loads from data on page itself?
pretty cool. Didn't know this was a thing.

Reply:
https://github.com/dimagi/commcare-hq/pull/25623/commits/e9694c9993c29186655c19c49de1de89877eef68

@orangejenny i made this change. Its unable to load the data though. 
i think `hqwebapp/js/toggles.js` JS is not loaded on default. I tried adding it on the page but it still fails to even load. can you help with this?

Reply:
It shouldn't add extra work, the toggles_dict is provided to all pages as initial page data: https://github.com/dimagi/commcare-hq/blob/10468bc9e93e09a143f8cbfd8dbb8ec6638f5ce6/corehq/apps/hqwebapp/templates/hqwebapp/base.html#L364

The example code was bad, I forgot this page uses requirejs, so it can't use `hqImport` (there's also a test failing because of this). Does it work if you add `hqwebapp/js/toggles` to the dependency list at the top and then reference that module down here?

Reply:
@orangejenny it does not.

Reply:
What does failing to load look like? With the code as it is, I do get an error about hqwebapp/js/toggles, not being loaded, but when I move the toggles dependency to the top and add a `console.log` like this, it correctly outputs the value of the flag: https://github.com/dimagi/commcare-hq/compare/mk/app-profiles-improv...jls/mk/app-profiles-improv?expand=1

Reply:
sorry , my bad.. i did include on top but didn't change the usage of calling `toggleEnabled` on the import. Apologies.

Reply:
works now, updated. 
https://github.com/dimagi/commcare-hq/pull/25623/commits/4f687f5ed8fc758625c661e31ead251da26f6c2d